metrics/web/lib/base/presentation/widgets/info_dialog.dart

```diff
@@ -119,11 +119,11 @@ class InfoDialog extends StatelessWidget {
             ),
             Align(
               alignment: Alignment.topRight,
-              child: TappableArea(
-                onTap: () => Navigator.of(context).pop(),
-                builder: (context, isHovered, child) => child,
-                child: Padding(
-                  padding: closeIconPadding,
+              child: Padding(
```

Comment:
Would it be better to use the `Container` widget instead of `Align` and `Padding` here?